components/brave_shields/browser/brave_shields_util.cc

```diff
@@ -359,6 +363,10 @@ void SetFingerprintingControlType(HostContentSettingsMap* map,
       ContentSettingsType::BRAVE_FINGERPRINTING_V2, content_setting);
 
   RecordShieldsSettingChanged(local_state);
+  if (url.is_empty()) {
```

Comment:
just curious why not put it into `DefaultBraveShieldsHandler::SetFingerprintingControlType` ?

Reply:
it seems that Android has it's own preference bridge, so I thought I'd add it here so that the reporting works on all platforms